### PR TITLE
Drop Python 3.4, add 3.7

### PR DESCRIPTION
**Pull request by mgedmin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-    - 3.4
     - 3.5
     - 3.6
-    # Force a newer PyPy on the old 'precise' CI image.
-    # After September 2017 this should be the default and the version
-    # pin can be removed.
-    - pypy-5.6.0
-    - pypy3.5-5.8.0
+    - 3.7
+    - pypy2.7-5.0.0
```
Comment (jamadden) [inline .travis.yml]: This version doesn't seem to be installable. Typo?

File: CHANGES.rst [hunk 2]
```diff
@@ -5,7 +5,9 @@
 4.3.2 (unreleased)
```
Comment (jugmac00) [inline CHANGES.rst]: When dropping Python 3.4, @icemac recommended to bump the version to a new major number, as for instance Debian 8 comes with Python 3.4 and is still supported (until 2022-06).

File: .travis.yml (hunk 1 above)
Comment (mgedmin) [reply .travis.yml]: Typo.

File: CHANGES.rst (hunk 2 above)
Comment (mgedmin) [reply CHANGES.rst]: Thank you for reminding me.

**Pull request merged**